docs/pages/guides/how-to/accounts/use-metamask-account.mdx

```diff
@@ -15,43 +15,43 @@ You can use both accounts with permissionless.js.
 
 ## Installation
 
-We will be using MetaMask's official SDK to create a smart account.
+Use MetaMask's official SDK to create a smart account.
 
 :::code-group
 
 ```bash [npm]
-npm install permissionless viem @metamask/delegation-toolkit
+npm install permissionless viem @metamask/smart-accounts-kit
 ```
 
 ```bash [yarn]
-yarn add permissionless viem @metamask/delegation-toolkit
+yarn add permissionless viem @metamask/smart-accounts-kit
 ```
 
 ```bash [pnpm]
-pnpm install permissionless viem @metamask/delegation-toolkit
+pnpm install permissionless viem @metamask/smart-accounts-kit
 ```
 
 ```bash [bun]
-bun install permissionless viem @metamask/delegation-toolkit
+bun install permissionless viem @metamask/smart-accounts-kit
 ```
 
 :::
 
-## Delegator Account
+## Delegator account
```

Comment:
There's a section above for Delegator Account, we should update it to mention what Delegator Account is, and why it needs to be a MetaMask Smart Accounts. Similar to our concepts page.